tests/player.test.js

```diff
@@ -68,19 +68,21 @@ describe('Player:', function () {
   it('like and dislike a video as an anonymous user @watch', () => {
     assertUserIsNotLoggedIn(browser)
     browser.url('http://localhost:3000/play/12345?playlist=98765')
-    browser.pause(2000)
+    browser.pause(1000)
```

Comment:
As I and @jellegerbrandy can attest to , `browser.pause` is evil. Is there a reason it is 100% needed here? I think using the `browser.waitUntil` will ultimately be more reliable on `CircleCI`

Reply:
ok!